### PR TITLE
Feature/creating order entity

### PR DESCRIPTION
**Pull request by Ramchy-J**
-Creating order entity
-Extending base to order
-Updating readme

### CODE REVIEW

File: entities/Order.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package entities;
+
+import constants.Status;
+
+import java.time.Instant;
+import java.util.Date;
```
Comment (eapg) [inline entities/Order.java:6]: Use `Instant` instead

File: entities/Order.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import java.util.List;
+
+public class Order extends Base {
+    //Attributes
+
+    private String customerName;
+    private List<Product> items;
+    private Double totalAmount;
+    private Date orderDate;
```
Comment (eapg) [inline entities/Order.java]: Use `Instant` instead

File: entities/Order.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    public Date getOrderDate() {
+        return orderDate;
+    }
+
+    public void setOrderDate(Date orderDate) {
+        this.orderDate = orderDate;
+    }
+}
```
Comment (eapg) [inline entities/Order.java]: add new line

File: entities/Order.java (hunk 2 above)
Comment (Ramchy-J) [reply entities/Order.java]: Done. order Date removed cuz there is an attribute to this in the base entity

File: entities/Order.java (hunk 3 above)
Comment (Ramchy-J) [reply entities/Order.java]: done

**Review by eapg — APPROVED** [review]

File: entities/Order.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    public void setItems(List<Product> items) {
+        this.items = items;
+    }
+
+    public void addItems(Product item){
+        items.add(item);
+        totalAmount += item.getPrice();
+    }
```
Comment (reynoldmorel) [inline entities/Order.java]: You don't need to dop this here

File: entities/Order.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+    //Methods
+
+    public String getCustomerName() {
+        return customerName;
+    }
+
+    public void setCustomerName(String customerName) {
+        this.customerName = customerName;
+    }
```
Comment (reynoldmorel) [inline entities/Order.java]: You might need more information about the customer, so either remove this or refer to a customer entity instead

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: entities/Order.java (hunk 4 above)
Comment (Ramchy-J) [reply entities/Order.java]: removed

File: entities/Order.java (hunk 5 above)
Comment (Ramchy-J) [reply entities/Order.java]: Referred to customer entity, It will be the next one to be created.

File: entities/Order.java [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+    //Methods
+
+    public String getCustomerInfo() {
+        return customerInfo;
+    }
+
+    public void setCustomerInfo(String customerInfo) {
+        this.customerInfo = customerInfo;
+    }
```
Comment (reynoldmorel) [inline entities/Order.java]: Looks like you still have customer info?

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: entities/Order.java (hunk 6 above)
Comment (Ramchy-J) [reply entities/Order.java]: Modified to Customer

**Review by reynoldmorel — APPROVED** [review]

**Pull request merged**